domain/entity/datastore.go

```diff
@@ -0,0 +1,37 @@
+package entity
+
+import (
+	"time"
+
+	"github.com/utahta/momoclo-channel/lib/config"
+	"google.golang.org/appengine/datastore"
+)
+
+type CreateTimestamper interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type CreateTimestamper should have comment or be unexported